src/notation/view/widgets/pagesettings.cpp

```diff
@@ -159,21 +162,21 @@ void PageSettings::updateValues()
 
     blockSignals(true);
 
-    const char* suffix;
+    muse::TranslatableString suffix;
     double singleStepSize;
     double singleStepScale;
     if (mm) {
-        suffix = "mm";
+        suffix = muse::TranslatableString("PageSettingsBase", "mm");
```

Comment:
Context should be "global", that way il also will benefit from the existing translations, will change.